### ISSUE TITLE
inside docker container, bazel cannot build sgx hardware mode 

### ISSUE THREAD

**Issue by Ruide** [google/asylo#28]
Hi,

I am building asylo using hardware mode with the following command:

docker run -it --rm \
	--device=/dev/isgx \
	-v /var/run/aesmd/aesm.socket:/var/run/aesmd/aesm.socket \
    -v bazel-cache:/root/.cache/bazel \
    -v "${MY_PROJECT}":/opt/my-project \
    -w /opt/my-project \
    gcr.io/asylo-framework/asylo \
    bazel run --config=sgx //hello_world -- --names="${NAMES}"

However, it says several http archive package cannot be found. Can someone help?

I have attached the full error log when i use the interactive mode to build as following.

rd@rd-desktop:~/Desktop/asylo-docker$ sudo docker run -it --rm     -v bazel-cache:/root/.cache/bazel     -v "${MY_PROJECT}":/opt/my-project     -w /opt/my-project     --device=/dev/isgx -v /var/run/aesmd/aesm.socket:/var/run/aesmd/aesm.socket     gcr.io/asylo-framework/asylo
root@73f53a8ff77f:/opt/my-project# bazel run --config=sgx //hello_world -- --names="${NAMES}"
Starting local Bazel server and connecting to it...
ERROR: /root/.cache/bazel/_bazel_root/b6bc0828d5b43c7085b650e0698bf388/external/com_google_asylo/asylo/platform/arch/BUILD:273:1: error loading package '@com_google_asylo//asylo/grpc/auth/core': Unable to load package for '@io_bazel_rules_go//go:def.bzl': The repository '@io_bazel_rules_go' could not be resolved and referenced by '@com_google_asylo//asylo/platform/arch:trusted_fork'
ERROR: /root/.cache/bazel/_bazel_root/b6bc0828d5b43c7085b650e0698bf388/external/com_google_asylo/asylo/platform/arch/BUILD:273:1: error loading package '@com_google_asylo//asylo/grpc/auth/core': Unable to load package for '@io_bazel_rules_go//go:def.bzl': The repository '@io_bazel_rules_go' could not be resolved and referenced by '@com_google_asylo//asylo/platform/arch:trusted_fork'
ERROR: Analysis of target '//hello_world:hello_world' failed; build aborted: error loading package '@com_google_asylo//asylo/grpc/auth/core': Unable to load package for '@io_bazel_rules_go//go:def.bzl': The repository '@io_bazel_rules_go' could not be resolved
INFO: Elapsed time: 5.368s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (54 packages loaded, 2785 targets \
FAILED: Build did NOT complete successfully (54 packages loaded, 2785 targets \
configured)
    currently loading: @com_google_asylo//asylo/grpc/auth/core

Thank you!

oh, i see the quick fix in user group by adding the following to workspace.

load("@com_google_asylo//asylo/bazel:asylo_deps.bzl", "asylo_go_deps")
asylo_go_deps()

**Comment by KeithMoyer** [comment]
@Ruide, this looks like the same issue as reported on the asylo-users mailing list (https://groups.google.com/d/msg/asylo-users/0qMD6TnoKQU/YEoKqKU-CQAJ). The solution is to add
```
load("@com_google_asylo//asylo/bazel:asylo_deps.bzl", "asylo_go_deps")
asylo_go_deps()
```
to the `WORKSPACE` file. We have addressed this issue at HEAD (https://github.com/google/asylo/commit/4098317c751197d975832719d8a8b515719ea74c) and it will be fixed in our next release. However, the examples tarball points at the latest release and needs these additional runs for now.

**Comment by KeithMoyer** [comment]
Oh, I see you edited your initial post that you already found the solution! :-D

Sorry for the trouble.

**Comment by KeithMoyer** [comment]
This should be fixed in the latest release (0.4.0).
